webapp/security/views.py

```diff
@@ -100,6 +100,9 @@ def notices():
     total_pages = ceil(total_results / page_size)
     offset = page * page_size - page_size
 
+    if page < 1 or 1 < page > total_pages:
```

Comment:
The first part of the second condition looks a bit redundant. Because you are using `or`, it means the number is greater than 1, so the second condition can just be `page > total_pages` without `1 < page`, right?